### ISSUE TITLE
답변 마지막에 나오는 value값의 의미가 무엇인지 알려줄 수 있으신가요?

### ISSUE THREAD

**Issue by sociengineer** [nawnoes/WellnessConversation-LanguageModel#10]
관련 코드 부분이 다음과 같은 것 같습니다.

output = model(**data)

    logit = output
    softmax_logit = nn.Softmax(logit).dim
    softmax_logit = softmax_logit[0].squeeze()

    max_index = torch.argmax(softmax_logit).item()
    max_index_value = softmax_logit[torch.argmax(softmax_logit)].item()

결과 값에 인덱스 뒤에 max_index_value값이 나오는데 이것은 그 카테고리로 구분된 가장 최고 값을 의미하는 것일까요?
어떤 값으로 어떻게 계산 된 것인지 설명을 해주시면 정말 좋겠습니다.

감사합니다.

**Comment by nawnoes** [comment]
안녕하세요
KoBERT와 KoELECTRA로 동일하므로 kobert-wellness-qa.py 기준으로 말씀드리겠습니다.

`
logit = output # 모델 결과 출력
softmax_logit = nn.Softmax(logit).dim # Softmax 취한후 nn.Softmax에서 dim으로 결과 tensor 받기
softmax_logit = softmax_logit[0].squeeze() # 결과 텐서가 tuple로 [0]으로 값 받아 squeeze()로 (1,359)-> (359)으로 변경

max_index = torch.argmax(softmax_logit).item() # Softmax 값 중 가장 큰 값의 인덱스를 받음
max_index_value = softmax_logit[torch.argmax(softmax_logit)].item() # 가장 큰 값에서 softmax_logit의 값 받음
 `

위 의도로 작성 하였는데, 추가로 필요하신 부분 있으시면 말씀해주세요~

**Comment by nawnoes** [comment]
제가 잘 못 사용한건지 nn.Softmax 의 결과 가 0~1 사이의 확률값으로 나오지 않아 
위에 부분 추가로 수정하였습니다.

```
    logit = output[0]
    softmax_logit = torch.softmax(logit,dim=-1)
    softmax_logit = softmax_logit.squeeze()

    max_index = torch.argmax(softmax_logit).item()
    max_index_value = softmax_logit[torch.argmax(softmax_logit)].item()
```

**Comment by sociengineer** [comment]
친절히 알려주셔서 대단히 감사합니다~
